homeassistant/components/sensor/srp_energy.py

```diff
@@ -2,7 +2,7 @@
 Platform for retrieving energy data from SRP.
 
 For more details about this platform, please refer to the documentation
-https://home-assistant.io/components/energy.srp/
+https://home-assistant.io/components/energy.srp_energy/
```

Comment:
I think the url should be this:
`https://home-assistant.io/components/sensor.srp_energy/`

When creating the docs, we can tag the page with the energy category, in the front matter.